mobile/components/UserEntry.tsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import { FontAwesome, Ionicons } from "@expo/vector-icons";
+import { TouchableOpacity, StyleSheet, View, Text } from "react-native";
+import IconButton from "./IconButton";
+
+const UserEntry = (props: any) => {
+  return (
+    <TouchableOpacity onPress={undefined} style={styles.userEntryContainer}>
+      <View style={styles.userEntry}>
+        <FontAwesome name="user-circle" size={24} color="grey" />
+        <Text style={styles.userLogText}>User Name</Text>
```

Comment:
make the user name a prop you can pass in -- no user name is specified default to "no username found"